staging/src/k8s.io/apiserver/pkg/admission/plugin/validatingadmissionpolicy/controller_reconcile.go

```diff
@@ -423,6 +438,26 @@ func (c *policyController) reconcilePolicyBinding(namespace, name string, bindin
 	return nil
 }
 
+func (c *policyController) reconcilePolicyStatus(namespace, name string, definition *v1alpha1.ValidatingAdmissionPolicy) error {
+	if definition != nil && definition.Status.ObservedGeneration < definition.Generation {
+		st := c.calculatePolicyStatus(definition)
+		newDefinition := new(v1alpha1.ValidatingAdmissionPolicy)
+		*newDefinition = *definition // do not mutate the object directly, make a shallow copy
+		newDefinition.Status = *st
+		_, err := c.client.AdmissionregistrationV1alpha1().ValidatingAdmissionPolicies().UpdateStatus(c.context, newDefinition, metav1.UpdateOptions{})
```

Comment:
if this gets a forbidden error, should it still retry? returning an error requeues the policy for sync

Reply:
Not really. Requeueing is a bad idea, especially for non kube-apiserver. Updating to change this behavior.

Reply:
this is logging forbidden errors now, but still returning a non-nil error which will still requeue

Reply:
Good point. No need to specially handle the forbidden case. We cannot let type checking alter the behavior of the controller anyway. Updated.